modules/nw-multus-ipam-object.adoc

```diff
@@ -13,41 +16,22 @@ ifeval::["{context}" != "configuring-macvlan"]
 :json:
 endif::[]
 
-:type: pass:q[Specify `static` to configure the plug-in to manage IP address \
-assignment. Specify `DHCP` to allow a DHCP server to manage IP \
-address assignment. You cannot specify any additional parameters if you \
-specify a value of `DHCP`.]
-
-:addresses: pass:q[that define IP addresses to assign to the virtual \
-interface. Both IPv4 and IPv6 IP addresses are supported.]
-
-:address: pass:q[A block of IP addresses specified in CIDR format to assign \
-to Pods on a worker node.]
-
-:address-gateway: pass:q[The default gateway to route egress network traffic to.]
-
-:routes: pass:q[describing routes to configure inside the Pod.]
-
-:route: pass:q[The IP address range in CIDR format.]
-
-:route-gateway: pass:q[The gateway to use to route network traffic to.]
-
-:dns: pass:q[The DNS configuration. Optional.]
-
-:nameservers: pass:q[one or more IP addresses for to send DNS queries to.]
-
-:domain: pass:q[The default domain to append to a host name. For example, if the \
-domain is set to `example.com`, a DNS lookup query for `example-host` will be \
-rewritten as `example-host.example.com`.]
-
-:search: pass:q[An array of domain names to append to an unqualified host name, \
-such as `example-host`, during a DNS lookup query.]
-
 [id="nw-multus-ipam-object_{context}"]
 = Configuration for ipam CNI plug-in
 
 The IP address management (IPAM) CNI plug-in manages IP address assignment for
-other CNI plug-ins.
+other CNI plug-ins. You can configure ipam for either static IP address
```

Comment:
s/ipam/IPAM?

Reply:
@mburke5678, based on comments in #forum-multus, the concept is IPAM, but the actual plug-in is just _ipam_. It's the same for the others. It's nonetheless jarring to read.